# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/hostlibc.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We do not want to build dietlibc twice for cross builds - just to get the
tiny wrapper for the building host ...

  - Rene Rebe <rene@exactcode.de>

--- dietlibc/diet.c	2000-01-01 00:00:00.000000000 +0100
+++ dietlibc-fixed/diet.c	2005-01-30 16:53:10.181709691 +0100
@@ -3,10 +3,13 @@
 #include <fcntl.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <write12.h>
 
 #include "dietfeatures.h"
 
+int __write2 (const char* s) {
+  return write(2, s, strlen(s));
+}
+
 /* goal:
  *   when invoked as
  * "diet gcc -c t.c"
